lib/Controller/ChatController.php

```diff
@@ -264,6 +264,9 @@ public function receiveMessages($token, $lookIntoFuture, $limit = 100, $lastKnow
 		$limit = min(200, $limit);
 		$timeout = min(60, $timeout);
 
+		$sessionId = $this->session->getSessionForRoom($token);
```

Comment:
Is `$sessionId` guaranteed to never be `null` here or should it be checked similar to when pulling signaling messages?

Reply:
I added a null check now. I don't dare to fail the request, because it can be useful to get chat messages via api without having to join the room